ImperatorToCK3/Source/CK3/CK3World.cpp

```diff
@@ -4,15 +4,28 @@
 #include <filesystem>
 #include <fstream>
 namespace fs = std::filesystem;
-#include "../Imperator/Characters/Character.h"
+#include "../Imperator/Countries/Country.h"
 #include "../Imperator/Provinces/Province.h"
 #include "../Configuration/Configuration.h"
 #include <cmath>
 #include "Province/CK3Provinces.h"
 #include "Province/CK3ProvinceMappings.h"
+#include "Titles/Title.h"
 
 CK3::World::World(const ImperatorWorld::World& impWorld, const Configuration& theConfiguration, const mappers::VersionParser& versionParser): outputModName(impWorld.getSaveName())
 {
+	LOG(LogLevel::Info) << "*** Hello CK3, let's get painting. ***";
+	// Scraping localizations from Imperator so we may know proper names for our countries.
+	localizationMapper.scrapeLocalizations(theConfiguration, std::map<std::string, std::string>()); // passes an empty map as second arg because we don't actually load mods yet
```

Comment:
usually we store mods inside configuration itself.
theConfiguration->getMods() grabs the map of <modname, modpath>
but if you want to keep them separate, that's fine.

Reply:
I haven't added that yet, don't want to bloat the PR.